packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="flex justify-between items-center">
+    <div class="flex gap-2 mb-3 mt-2">
+      <img
+        v-if="workspaceInfo.logo"
+        :src="workspaceInfo.logo || ''"
+        alt="Workspace logo"
+        class="w-5 h-5 mt-0.5"
+      />
+      <div class="flex flex-col">
+        <h1 class="text-heading-lg">{{ workspaceInfo.name }}</h1>
+        <div class="text-body-xs text-foreground-2">
+          {{ workspaceInfo.description || 'No workspace description' }}
+        </div>
+      </div>
+    </div>
+    <div class="flex items-center gap-2">
+      <div
+        class="text-body-3xs bg-foundation-2 text-foreground-2 rounded px-3 py-1 font-medium select-none"
+      >
+        {{ projectCount || 0 }} Project{{ projectCount === 1 ? '' : 's' }}
+      </div>
+      <UserAvatarGroup
+        :users="team.map((teamMember) => teamMember.user)"
+        class="max-w-[104px]"
+      />
+      <FormButton color="outline">Invite</FormButton>
+      <WorkspaceHeaderActions />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { graphql } from '~~/lib/common/generated/gql'
+import type { WorkspaceInfo_WorkspaceFragment } from '~~/lib/common/generated/gql/graphql'
+
+graphql(`
+  fragment WorkspaceInfo_Workspace on Workspace {
```

Comment:
The fragment name is supposed to be based on the component name: WorkspaceHeader_Workspace